homeassistant/scripts/benchmark/__init__.py

```diff
@@ -315,6 +315,22 @@ async def json_serialize_states(hass):
     return timer() - start
 
 
+class Row(NamedTuple):
+    """Tuple for a row."""
+
+    event_type: EventType
+    event_data: str
+    time_fired: float
+    context_id: None
+    context_user_id: None
+    state: Any
+    domain: str
+    entity_id: str
+    attributes: str
+    state_id: int
+    old_state_id: int
+
+
 def _create_state_changed_event_from_old_new(
```

Comment:
This function is not used, but I think that there is a reason for it to be there in the benchmark script.